### DISCUSSION TITLE
为什么付费用户的公益额度也被清空了?

### DISCUSSION THREAD

**Discussion by Sunnnner** [BurnHair/docs#175] [技术支持]
<img width="416" alt="image" src="https://github.com/user-attachments/assets/06e945a0-69b2-4414-86d1-96e7f20cefcf">
<img width="660" alt="image" src="https://github.com/user-attachments/assets/34f439a5-d572-4c37-a3dc-40a69ff04642">

**Comment by Sunnnner** [comment]
是我的问题我的接口被刷爆了。。。

**Reply by BennyThink** [reply]
😱 那你可要检查下原因，有需要随时联系

**Reply by Sunnnner** [reply]
> 😱 那你可要检查下原因，有需要随时联系

好的
